[EDC-2483][EDC-2486] Get translated labels, handle default language

DanielM235 opened the pull request:
- translated labels are now defined from i18N export files
- use configuration language as default language

Story [EDC-2375][EDC-2376]
Epic [EDC-1796]

Commit:
[EDC-2483][EDC-2486] Get translated labels, handle default language

- translated labels are now defined from i18N export files
- use configuration language as default language

Story [EDC-2375][EDC-2376]
Epic [EDC-1796]

greg7859 commented on src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java:
```diff
@@ -87,7 +87,7 @@ public void mouseExited(MouseEvent e) {
         String url = "";
         try {
-            url = edcClient.getContextWebHelpUrl(mainKey, subKey, languageCode);
+            url = edcClient.getContextWebHelpUrl(mainKey, subKey, this.helpConfiguration.getLanguageCode());
```
languageCode is unused.
Remove it.

DanielM235 replied:
Done

greg7859 commented on src/test/java/fr/techad/edc/popover/swing/example/Main.java:
```diff
@@ -106,4 +112,22 @@ private static void createAndShowGUI() {
         f.setVisible(true);
     }
+
+    private static JComboBox<String> createLangSelector() {
```
Add this new option in the documentation.

DanielM235 replied:
Done

greg7859 reviewed (changes requested):
Need to clean the source and improve the documentation to explain how to integrate the multi langage support.

Commit:
[EDC-2483][EDC-2486] Get translated labels, handle default language

- Remove unused languageCode variable in IconButtonListener.java, since language is defined in the configuration
- set version to 2.5.1-SNAPSHOT
- add a translation section in readme file

Story [EDC-2375][EDC-2376]
Epic [EDC-1796]
